components/raftstore/src/store/peer.rs

```diff
@@ -925,6 +925,12 @@ where
     /// this peer has raft log gaps and whether should be marked busy on
     /// apply.
     pub last_leader_committed_idx: Option<u64>,
+
+    /// Used for recording the new created and uncampaigned regions by `Split`.
```

Comment:
Adding a bit more context
```suggestion
    /// Used to record uncampaigned regions, which are the new regions 
    /// created when a follower applies a split. If the follower becomes a 
    /// leader, a campaign is triggered for those regions.
```